pkg/istio-agent/agent.go

```diff
@@ -467,7 +467,7 @@ func (a *Agent) Run(ctx context.Context) (func(), error) {
 }
 
 func (a *Agent) initLocalDNSServer() (err error) {
-	// we dont need dns server on gateways
+	// we don't need dns server on gateways
```

Comment:
Please avoid refactoring in a PR with real changes.. it makes it hard to review

Reply:
> Please avoid refactoring in a PR with real changes.. it makes it hard to review

Sorry,  i got it